### PR TITLE
Webhook support for person.email

### PR DESCRIPTION
**Pull request by AdriVanHoudt**
Added webhook support for `person.email` lookup and added a test for it.
Also added a test for the empty phone test :wink:

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
 language: node_js
 node_js:
-  - "0.8"
```
Comment (3rd-Eden) [inline .travis.yml]: What was the reason for nuking `0.8`?

**Comment by 3rd-Eden** [comment]
LGTM except for the removal of 0.8

File: .travis.yml (hunk 1 above)
Comment (AdriVanHoudt) [inline .travis.yml]: It's old?

**Comment by AdriVanHoudt** [comment]
If you want I can enable it again?

File: .travis.yml (hunk 1 above)
Comment (3rd-Eden) [inline .travis.yml]: If it still works, there's no need to remove it.

**Comment by 3rd-Eden** [comment]
@AdriVanHoudt I'll add it back, thanks for the pull request. I'll kick out a new minor version in a bit.

**Comment by AdriVanHoudt** [comment]
Thanks! and I just pushed the 0.8 support to my fork :smile:

**Comment by 3rd-Eden** [comment]
0.8 seems to fail, i'm gonna nuke it

**Comment by AdriVanHoudt** [comment]
I got `Error: usage limits are exceeded` maybe the key used was used to much

**Comment by 3rd-Eden** [comment]
@AdriVanHoudt Yes, nothing I can do that unfortunately. Full contact doesn't have any keys that can be used for testing and I didn't really feel like mocking every single response up.

**Comment by AdriVanHoudt** [comment]
Yeah no prob, I know that some libs require the dev to use his own key, maybe that will help?

**Comment by 3rd-Eden** [comment]
@AdriVanHoudt I could surely do that, but that requires extra magic to make the tests work with Travis-CI. (Secret env variables etc) and you would still be limited to the API key.

**Comment by AdriVanHoudt** [comment]
@3rd-Eden true maybe advice devs in the readme to use their own keys with local testing?

**Comment by 3rd-Eden** [comment]
@AdriVanHoudt Moved the API key, added a dedicated test script for travis-ci so that should help prevent more failures in the future. And i'll update the docs in a bit.

**Comment by AdriVanHoudt** [comment]
@3rd-Eden cool! If I have time i'll look into adding webhook support for all the other endpoints to(where possible)

**Comment by 3rd-Eden** [comment]
@AdriVanHoudt I'm also more than happy to add you as contributor to the module if you want to. It would probably allow you to iterate a bit quicker. (As long as you don't break the existing tests ;-))

**Comment by AdriVanHoudt** [comment]
@3rd-Eden fine by me :smile: tbh I'm pretty new to the open source stuff but I'll do my best! (to not break everything :laughing: )

**Comment by 3rd-Eden** [comment]
E-mail send, if you need npm access as well, give me a poke.

**Comment by AdriVanHoudt** [comment]
I saw it thanks! Maybe for now not so you are required to review changes :wink:
